### PR TITLE
Itamar commit test

### PR DESCRIPTION
**Pull request by villigernu7**

### CODE REVIEW

File: GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	public Drivetrain(SpeedController leftGearbox, SpeedController rightGearbox, Encoder encoderLeft,
+			Encoder encoderRight) {
+		this.leftGearbox = leftGearbox;
+		this.rightGearbox = rightGearbox;
+		this.encoderLeft = encoderLeft;
+		this.encoderRight = encoderRight;
+	}
+
+	public void initDefaultCommand() {
+		// TODO bom
```
Comment (mantin2212) [inline GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java]: no words are needed. plz delete dis

**Review by mantin2212 — DISMISSED** [review]

File: GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+/**
+ *
+ */
+public class Drivetrain extends TankDrivetrain {
+
+	private SpeedController leftGearbox;
+	private SpeedController rightGearbox;
+	private Encoder encoderLeft;
+	private Encoder encoderRight;
+
```
Comment (TomerSitton) [inline GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java]: no need for the word gearbox here. we are not ... מכניקאים... YAK!
use leftSpeedController instead

File: GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+			Encoder encoderRight) {
+		this.leftGearbox = leftGearbox;
+		this.rightGearbox = rightGearbox;
+		this.encoderLeft = encoderLeft;
+		this.encoderRight = encoderRight;
+	}
+
+	public void initDefaultCommand() {
+	}
+
```
Comment (TomerSitton) [inline GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java]: add TODO to add default command

File: GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+	@Override
+	public PIDSource getLeftPIDSource() {
+		return encoderLeft;
+	}
+
+	@Override
+	public PIDSource getRightPIDSource() {
+		// TODO Auto-generated method stub
+		return encoderRight;
```
Comment (TomerSitton) [inline GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java]: remove comment

File: GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+public class Drivetrain extends TankDrivetrain {
+
+	private SpeedController leftGearbox;
+	private SpeedController rightGearbox;
+	private Encoder encoderLeft;
+	private Encoder encoderRight;
+
+	public Drivetrain(SpeedController leftGearbox, SpeedController rightGearbox, Encoder encoderLeft,
+			Encoder encoderRight) {
+		this.leftGearbox = leftGearbox;
```
Comment (TomerSitton) [inline GearWorksXO-2017/src/org/usfirst/frc/team2212/robot/subsystems/Drivetrain.java]: add TODO - check inverted for motors

**Review by TomerSitton — CHANGES_REQUESTED** [review]

**Review by TomerSitton — APPROVED** [review]

**Pull request merged**